src/mesh/simplemesh.jl

```diff
@@ -44,3 +44,26 @@ list of points and [`FullTopology`](@ref).
 """
 Base.convert(::Type{<:SimpleMesh}, m::Mesh) =
   SimpleMesh(vertices(m), topology(m))
+
+
+"""
+    isinside(point, mesh)
+
+Tells whether a `point` is inside a surface `mesh`. Must be a surface mesh with only triangles as elements. Points that coincide with vertices of the mesh or lies on the face of a triangle return true.
+"""
+function isinside(testpoint::Point{3,T}, mesh::Mesh{3,T}) where T
```

Comment:
Please rename `testpoint` to `point`. Also, do we need to specify the type parameters at this point?

What about:

```julia
function isinside(point::Point, mesh::Mesh)
```